testing/cups-epson-escpr/APKBUILD

```
@@ -0,0 +1,33 @@
+# $Id$
+# Maintainer: Michael Wyraz <michael@wyraz.de>
+# Adapated from https://aur.archlinux.org/packages/epson-inkjet-printer-escpr/
+#
+
+pkgname=cups-epson-escpr
+pkgver=1.6.29
+pkgrel=0
+pkgdesc="Epson Inkjet Printer Driver (ESC/P-R) for Linux"
+arch="all"
+url="http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX"
+license="GPL-2.0-or-later"
+depends="cups ghostscript"
+makedepends="cups-dev"
+options="!libtool !check" # Test suite is missing
```

!libtool is the default setting so isn't required

```suggestion
options="!check" # Test suite is missing
```